core/src/main/resources/application.conf

```diff
@@ -19,27 +19,35 @@ db {
 
 languagesConfig {
   haskell {
+  	repository = "hackage"
     repoIndexUrl = "http://hackage.haskell.org/packages/index.tar.gz"
+    packageUrl = "https://hackage.haskell.org/package/%1$s-%2$s/%1$s-%2$s.tar.gz"
     repoArchivePath = "./data/meta/haskell/index.tar.gz"
     repoPath = "./data/meta/haskell/"
     concurrentTasksCount = 30
   }
   rust {
+  	repository = "crates"
     repoIndexUrl = "https://github.com/rust-lang/crates.io-index/archive/master.zip"
+    packageUrl = "https://crates.io/api/v1/crates/%s/%s/download"
     repoArchivePath = "./data/meta/rust/archive.zip"
     repoPath = "./data/meta/rust/"
     concurrentTasksCount = 30
+    ignoreFiles = ["test-max-version-example-crate", "version-length-checking-is-overrated", "config.json", "archive.zip", ".git"]
```

Comment:
Why is config.json excluded? (Not saying it shouldn't be, just asking why)

Reply:
@neongreen I can't answer why is so =( but it was originally. I will remove it, thanks.